command/ca/rekey.go

```diff
@@ -166,7 +166,7 @@ flag.`,
 			cli.StringFlag{
 				Name: "pid-file",
 				Usage: `The <file> from which to read the process id that will be signaled after the certificate
-has been rekeyed. By default the the SIGHUP (1) signal will be used, but this can be configured with the **--signal**
+has been rekeyed. By default the SIGHUP (1) signal will be used, but this can be configured with the **--signal**
```

Comment:
I'm still getting comfortable w/ duplicate word detection, but overall, it's pretty good and I haven't had much pushback of late.